Content.Server/Administration/Commands/RejuvenateCommand.cs

```diff
@@ -98,6 +99,11 @@ public static void PerformRejuvenate(EntityUid target)
             {
                 EntitySystem.Get<DiseaseSystem>().CureAllDiseases(target, carrier);
             }
+
+            if (entMan.TryGetComponent<RottingComponent>(target, out var rotting))
+            {
+                entMan.RemoveComponent<RottingComponent>(target);
+            }
```

Comment:
```suggestion
            entMan.RemoveComponent<RottingComponent>(target);
```

remcomp checks for this already